AIM/AIM/Domain/Entities/StationInfo.h

```diff
@@ -0,0 +1,24 @@
+//
+//  StationInfo.h
+//  AIM
+//
+//  Created by Oskar Szydlowski on 04.03.2018.
+//  Copyright © 2018 Oskar Szydlowski. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface StationInfo : NSObject
+
+@property (nonatomic, strong) NSString *name;
+@property (nonatomic, strong) NSString *description;
+@property (nonatomic, strong) NSString *time;
+@property (nonatomic, strong) NSString *duration;
+@property (nonatomic, strong) NSString *presenter;
```

Comment:
`presenter` is a bit unfortunate name since we use it for the components managing the views. I would use some different name here (or a least add some suffix, like `presenterName`?)